### ISSUE TITLE
Project Dead? - Zero Updates in 8 Months

### ISSUE THREAD

**Issue by war59312** [ShankarBUS/ModernFlyouts#1153]
### ModernFlyouts Version

0.9.3.0

### Windows version

Windows 11 22H2

### OS build

22621.963

### Platform

x64

### Steps to reproduce

Is this project dead?

### ❌ Actual Behavior

Been zero updates for at least 8 months now.

**Comment by tuongdai252** [comment]
They're abandoning the current version to rewrite completely a new version. I think they're just not ready to publish it yet.

**Comment by ElitePheonix2009** [comment]
I asked them this through a discussion 3 months earlier, they said this -
"ModernFlyouts IS STILL MAINTAINED.
The reason for the release delay is due to a large rebuild of the app taking place which as taken a long time.
A lot of these delays are actually thanks to Microsoft, due to issues with the windows app sdk.
We hope to release 0.11 soon."

**Comment by war59312** [comment]
It's an open-source project and if that is the case, I'd expect to see code changes,

Never a good idea to just randomly one day dump a "new" code base.

At very least have another Branch. The 0.11-final branch hasn't been touched in 9 months.

Seeing is believing. For most, I am sure we are moving on. By the time this "hope to release 0.11" comes around, no one will care. Which hurts development efforts even more.

I for one appreciate the effort but at this point it seems the project is very much dead.

I posted this 3 weeks ago and nothing from development team. If someone cared enough, he/she would post something.

**Comment by hdadhich01** [comment]
Got an update from a dev, they have other university work to attend to. Here is a [current workaround](https://github.com/ModernFlyouts-Community/ModernFlyouts/issues/1161#issuecomment-1399208776) for now. Also, people have lives.

**Comment by ShankarBUS** [comment]
> It's an open-source project and if that is the case, I'd expect to see code changes,
> 
> Never a good idea to just randomly one day dump a "new" code base.
> 
> At very least have another Branch. The 0.11-final branch hasn't been touched in 9 months.
> 
> Seeing is believing. For most, I am sure we are moving on. By the time this "hope to release 0.11" comes around, no one will care. Which hurts development efforts even more.
> 
> I for one appreciate the effort but at this point it seems the project is very much dead.
> 
> I posted this 3 weeks ago and nothing from development team. If someone cared enough, he/she would post something.

![images (1).jpeg](https://user-images.githubusercontent.com/63704247/215307181-f314553c-e6c8-4ae8-b30c-b5abdb9308e6.jpeg)

I get your frustration.

I used to be the main developor this app.

I had to quit development for several reasons.

My teammates are trying their best to add features and fix bugs. The UI change they're doing requires a big rewrite and it will take a lot of time to figure out which is what. One guy learnt coding just to keep this alive.

And I couldn't aid any help to them from my side too.

All of them are really busy at this point.

And Microsoft's app solutions are not helping too. WinAppSDK is still sluggish, buggy, no support for transparent windows, etc.

Our app currently is built on ModernWpf/WPF.

Both of them are dead?

So, we had to move to another platform that is either WinUI 2/XAML Islands or WinAppSDK. Both of them have their pros and cons.

So, we are in a state of confusion what yo do. And our busy schedules aren't help either.

We will do our best.
We can't provide any ETA for the next update yet. We will let y'all know if there are any new plans or updates.

Thanks for staying and sorry too.
